### DISCUSSION TITLE
Log/pcap not loading

### DISCUSSION THREAD

**Discussion by Dapylad** [Security-Onion-Solutions/securityonion#13442] [2.4]
I imported pcap into security onion and it's not loading nor showing in my console.
Please what's the hiccups and what to do to get it done.![PXL_20240806_145241352.jpg](https://github.com/user-attachments/assets/728ab71f-fe65-49ad-a2de-f2bc31df3adf)

**Comment by reyesj2** [comment]
What type of install is this? 
Does the grid page show that all the services are running?
What are the specs of the virtual machine running Security Onion?

When you create a new discussion there are a few questions to fill out. This helps avoid asking the above questions or similar on every discussion post created https://github.com/Security-Onion-Solutions/securityonion/discussions/new?category=2-4

**Comment by Dapylad** [comment]
Import node installation
And yes all services are running, all green.
I imported pcap so i can hunt and investigate but it is not showing the
content of the pcap.

**Comment by Dapylad** [comment]
Hello Jorge,
Trust you are doing good
Still having certain challenges
Pcap imported not loading in dashboard nor alerts.

And i checked Grid, everything is ok and green state
Please how to rectify it

**Reply by cm-ops** [reply]
One thing to verify is the time frame of the imported PCAP, is the current date range the correct range to the traffic?

**Reply by Dapylad** [reply]
Yes it is.
I adjusted the date.

**Reply by Dapylad** [reply]
Version is 2.4.6
Installation is ISO image 
Configuration is an import node 
Machine spec above minimum requirements ( 8GB, 512GBROM, 4 CPU).
Yet the pcao not loading.
Imported via Grid, didn't load
Imported via CLI , same response.

**Comment by dougburks** [comment]
Are you installing in a virtual machine or on a dedicated computer?

What type of storage (NVME, SSD, or rotational)?

Have you tried the latest version 2.4.90?
https://blog.securityonion.net/2024/07/security-onion-2490-now-available.html

**Comment by Dapylad** [comment]
Yes i installed in a virtual machine ( Oracle virtualbox)

Storage is HDD

Currently using 2.4.6
Tried to upgrade to 2.4.9 using soup, it's not upgrading.
Always gives error with 1 update failed

**Reply by dougburks** [reply]
> Yes i installed in a virtual machine ( Oracle virtualbox)

What are the specs of the host machine (CPU, RAM, etc.)?

Are you running other virtual machines or apps at the same time?

> Storage is HDD

Elasticsearch typically recommends SSD or NVME. Rotational media might be OK for a simple import node, so I'm guessing the most likely culprit is that your host machine is oversubscribed.

> Currently using 2.4.6
Tried to upgrade to 2.4.9 using soup, it's not upgrading.
Always gives error with 1 update failed

You could try a fresh installation using the 2.4.90 ISO image but you'll probably still run into issues if your host machine is oversubscribed.

**Comment by Dapylad** [comment]
Machine is HDD
8 CPU
512GbROM
8 GB RAM
Not running another virtual machine.

I make sure host not over used

Downloaded 2.4.9 iso, installing at the moment
I'll keep you posted

Thank you for your response

**Reply by dougburks** [reply]
If your host machine has 8GB RAM, how much are you assigning to the Security Onion VM?

**Comment by Dapylad** [comment]
I assigned 5Gb

**Reply by dougburks** [reply]
With 8GB of physical RAM and 5GB assigned to the Security Onion VM, it's quite possible that the host machine is overloaded. The Windows OS and apps could be using more than 3GB RAM which could lead to swapping and general performance issues.

For best results, please consider one of the following recommendations if possible:
- increase physical RAM in host machine
- find another machine that can be dedicated to Security Onion without virtualization

Also keep in mind the hardware requirements and best practices:
https://docs.securityonion.net/en/2.4/hardware.html
https://docs.securityonion.net/en/2.4/best-practices.html
